python/lsst/cp/pipe/utils.py

```diff
@@ -1032,33 +1128,47 @@ def compute_ratio_model(nodes, group_indices, pars, pd, mu, return_spline=False)
             Array of node positions.
         group_indices : `list` [`np.ndarray`]
             List of group indices, one array for each group.
+        par_indices : `dict`
+            Dictionary showing which indices in the pars belong to
+            each set of fit values.
         pars : `np.ndarray`
             Parameter array, with spline values (one for each node) followed
-            by proportionality constants (one for each group.)
+            by proportionality constants (one for each group.), followed by
+            (optionally) one offset O.
```

Comment:
Similar comment here: I don't think these docstrings were updated after the fit parameters were extended.